server/src/ServerGame.cpp

```diff
@@ -49,37 +48,16 @@ void ServerGame::handleInitConnection(unsigned int client_id) {
 
 void ServerGame::handleClientActionInput(unsigned int client_id, ClientToServerPacket& packet)
 {
-    // for testing now 
-    // std::printf("client(%d): W(%d) A(%d) S(%d) D(%d) Jump(%d) | yaw(%f) pitch(%f)\n", client_id, packet.requestForward, packet.requestLeftward, packet.requestBackward, packet.requestRightward, packet.requestJump, packet.yaw, packet.pitch);
-
     // Book keeping for each client (so they can render each other)
-    yaws[client_id] = packet.yaw;
-    pitches[client_id] = packet.pitch;
 
-    // Update player position (no pitch here, cuz you can't fly)
+    // Update player's moving direction (no pitch here, cuz you can't fly)
     glm::vec3 forward_direction;
     forward_direction.x = cos(glm::radians(packet.yaw));
-    forward_direction.y = velocities[client_id].y;  // to handle jump (todo: high V.y when space pressed then decrement with gravity)
+    forward_direction.y = 0;
     forward_direction.z = sin(glm::radians(packet.yaw));
     forward_direction = glm::normalize(forward_direction);
```

Comment:
nit: why is forward_direction calculated here (since most update logic is moved to systems' update(), shouldn't this too?)